File: CursoCsharpUdemy/exer/outros/Exer21.cs
/*
*   21) Faça um programa que leia um número inteiro e mostre uma mensagem na tela
*   indicando se este número é positivo ou negativo. Pare a execução do programa
*   quando o usuário requisitar.
*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer21
    {
        static void Main(string[] args)
        {
            int number;

            Console.WriteLine("Informe um número inteiro: ");
            number = int.Parse(Console.ReadLine());

            if (number >= 0)
            {
                Console.WriteLine("Número positivo");
            }
            else
            {
                Console.WriteLine("Número negativo");
            }
        }
    }
}

File: CursoPython3/exer/lista_64_Exer/exer21.py
"""
21) Faça um programa que leia um número inteiro e mostre uma mensagem na tela indicando se este número é 
positivo ou negativo. Pare a execução do programa quando o usuário requisitar. 
"""

num = int(input("Informe um número inteiro:"))

if (num >= 0):
    print(f"O número {num} é positivo.")
else:
    print(f"O número {num} é negativo.")
